decentraland/unity-renderer
# chore: move the shader prewarm to the end of the loading screen

## What does this PR change?

Moved the shader prewarm from startup to the end of the loading screen, increasing the UX. 

## How to test the changes?

This should PR should: 
- Remove the screen freeze of several seconds after selecting to start with wallet or guest
- Show a message at the end of the loading that explains that the shaders are getting prewarmed
- Ensure that the shaders are being prewarmed just once (teleporting after initial loading)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
